app/assets/javascripts/search.js

```diff
@@ -121,7 +121,7 @@
 
       if (GOVUK.analytics !== undefined &&
           GOVUK.analytics.trackEvent !== undefined &&
-          $searchResults.length) {
+          (searchResultData['urls'].length || searchResultData['suggestion'])) {
```

Comment:
This would be more JavaScript'y as `(searchResultData.urls.length || searchResultData.suggestion)`. If the key is a string it you don't need to use the array syntax and can just use dot-notation.